table.go

```diff
@@ -504,58 +517,62 @@ func (t *Table) center(i int, isFirstRow, isLastRow bool) string {
 			return t.syms[symEW]
 		}
 		if isFirstRow {
-			return t.syms[symES]
+			return t.syms[symESb]
```

Comment:
`t.syms[symESb]` must we use map, can we not have a theme struct?

Reply:
It's actually a slice, not a map. Here are my thoughts on the trade-offs:

1. Slice allows doing arithmetic on the offsets. So currently you can get from a non-boundary symbol to the corresponding boundary symbol by adding 11 to the symbol constant. I thought I would need that, but it turns out I don't.
2. There are a number of places in this change here where I compute the index in one place, and then look it up in a another. That would no longer work with a struct, so we would always have to directly resolve to the symbol itself.
3. If we add any form of character-changing API, that API might benefit from being able to refer to symbol roles as opposed to the current assignment for a given role. Maybe a pointer into a struct can serve that purpose, but it's less obvious.